### PR TITLE
Practice 1

### PR DESCRIPTION
**Pull request by duylong13**

### CODE REVIEW

**Comment by huynguyenbich-agilityio** [comment]
@duylong13 Please add Vuong Nguyen to your Github repo and reassign the PR to him.

File: Practice1/src/blog-2.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.heading-1 {
```
Comment (vuongnguyen-agilityio) [inline Practice1/src/blog-2.css:1]: Don't naming class by number

File: Practice1/src/blog-2.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    position: absolute;
+    color: white;
+    width: 100%;
+    text-align: center;
+    margin-top: -600px;
+}
+
+.group-14 {
```
Comment (vuongnguyen-agilityio) [inline Practice1/src/blog-2.css:9]: Same above

File: Practice1/src/blog-2.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    position: relative;
+    margin: 50px 40px 0 10px;
+    left: 0;
+    top: 0;
+    width: 200px;
+    border: 0px;
+    box-sizing: border-box;
+}
+
+.mask img {
```
Comment (vuongnguyen-agilityio) [inline Practice1/src/blog-2.css:25]: Don't style for tag, add class instead

File: Practice1/src/blog-2.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    width: 198px;
+}
+
+.div-common {
```
Comment (vuongnguyen-agilityio) [inline Practice1/src/blog-2.css:29]: Pls rename this class because is not meaningful

File: Practice1/src/blog-2.css [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+    background-color: white;
+}
+.top-medical {
```
Comment (vuongnguyen-agilityio) [inline Practice1/src/blog-2.css:32]: Missing new line with line 31

File: Practice1/src/blog-2.css [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+    font-size: 22px;
+    text-align: center;
+}
+
+.top-medical p{
+    font-size: 15px;
+}
+
+.lorem {
```
Comment (vuongnguyen-agilityio) [inline Practice1/src/blog-2.css:41]: Not meaningful. Please rename it

File: Practice1/src/blog-2.css [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    font-size: 10px;
+    text-align: center;
+}
+
+.common {
+    position: relative;
+    padding-top: 7px;
+    display: flex;
+    justify-content: space-around;
+}
```
Comment (vuongnguyen-agilityio) [inline Practice1/src/blog-2.css:56]: EOL error. Please fix.

File: Practice1/src/index.html [hunk 8]
```diff
@@ -0,0 +1,328 @@
...
+    <link rel="stylesheet" href="header.css">
+    <link rel="stylesheet" href="blog.css">
+    <link rel="stylesheet" href="blog-2.css">
+    <title>HEALY</title>
+</head>
+
+<body>
+    <header>
+        <div class="nav">
+            <img class="nav-img" src="./assets/images/shutterstock_318359918.png" alt="" width="100%" , height="100%">
```
Comment (vuongnguyen-agilityio) [inline Practice1/src/index.html:20]: Move all style to CSS. Please remove width, height in HTML file

File: Practice1/src/index.html [hunk 9]
```diff
@@ -0,0 +1,328 @@
...
+                        </div>
+                        <div class="common-2">
+                            Read more
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <!-- <section>
```
Comment (vuongnguyen-agilityio) [inline Practice1/src/index.html:167]: Remove unused code here

**Comment by vuongnguyen-agilityio** [comment]
Please update the PR following:
1. Remove unused code
2. Remove .cache, disk
3. Add cache, disk to .gitignore
4. Fix error: No new line at the end of line in git
5. Fix all code comments

**Pull request closed without merging**